server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixAccountEntity.java

```diff
@@ -0,0 +1,30 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import jdk.jfr.Enabled;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "netflix_account_entity")
+public class NetflixAccountEntity {
+
+    @Id
+    @Column(
+            name = "id"
+    )
+    private Long id;
+
+    @Column(
+            name = "nb_devices"
+    )
+    private int nb_devices;
+
+    @Column(
+            name = "typeAccount"
+    )
+    private String typeAccount;
+
+    @OneToOne
+    private NetflixUserEntity uuid_user;
```

Comment:
Ici tu représente une relation `@OneToOne` et pas un UUID donc le nom de ton attribut est pas bon , et en plus il est en `snake_case` donc ici tu devrait avoir : 
```suggestion
    private NetflixUserEntity netflixUserEntity;
```